DEVELOPMENT_GUIDE.md

```diff
@@ -328,6 +328,18 @@ conventions are best practices that we have learnt over time.
     comments.
     
     
+### Dependency Updates
+
+Please update all the required files if the changes involve a version update on a dependency or to include a new dependency. The requirements files are located inside the `requirements folder`.
```

Comment:
The contents looks good, could we structure it such there are 3 parts, base, reproducible and installer?

Reply:
Yep makes sense! I can break down the sections further for this to be clearer